src/views/TournamentDetail.vue

```diff
@@ -1,573 +1,96 @@
 
-const knockout_match_results = (match: KnockoutMatch) => {
-  const match_results = [] as Record<string, string | number | boolean | undefined>[];
-  match.teams.forEach((team) => {
-    const data = {} as Record<string, string | number | boolean | undefined>;
-    data.name = get_validated_team_by_id(team)?.name;
-    data.score = match.score[team];
-    data.is_winner = is_winning_team(match, team);
-    match_results.push(data);
-  });
+interface TournamentDetailSection {
+  title: string;
+  is_available: boolean;
+  is_selected: boolean;
+}
 
-  return match_results;
-};
+const sections = computed<Record<string, TournamentDetailSection[]>>(() => ({
+  info: { title: 'Informations', is_available: true },
+  teams: { title: 'Équipes', is_available: true },
+  groups: { title: 'Poules', is_available: tournament.value?.groups.length > 0 || false },
+  swiss: { title: 'Système Suisse', is_available: tournament.value?.swissRounds.length > 0 || false },
+  brackets: { title: 'Arbres', is_available: tournament.value?.brackets.length > 0 || false },
+  planning: { title: 'Planning', is_available: true },
+  rules: { title: 'Règlement', is_available: true },
+}));
 
-const swiss_match_results = (matchs : SwissMatch[]) => {
-  const round_matchs = groupBy(matchs, 'round_number');
-  const res = {} as Record<string, Record<string, string | Record<string, string | number | boolean | undefined>[]>[]>;
-  Object.entries(round_matchs).forEach(([round, roundMatchs]) => {
-    res[round] = [] as Record<string, string | Record<string, string | number | boolean | undefined>[]>[];
-    roundMatchs.forEach((match) => {
-      const tmp = {} as { teams: Record<string, string | number | boolean | undefined>[]; status: string };
-      tmp.teams = [];
-      tmp.status = match.status;
-      match.teams.forEach((team) => {
-        const data = {} as { name: string | undefined; score: number; is_winner: boolean };
-        data.name = get_validated_team_by_id(team)?.name;
-        data.score = match.score[team];
-        data.is_winner = is_winning_team(match, team);
-        tmp.teams.push(data);
-      });
-      res[round].push(tmp);
-    });
-  });
+const router = useRouter();
+const route = useRoute();
 
-  return res;
-};
+const selected_section = ref<string>(route.path.split('/').at(-1));
 
-const router = useRouter();
 try {
   await getTournamentFull(props.id);
   getTournamentTeams();
 } catch (err: unknown) {
   router.go(-1);
 }
-
-if (props.section !== undefined && props.section.s in sections) {
-  sections[props.section.s][0] = true;
-  drop_label.value = props.section.s;
-  trans.value = `translateX(calc(-100vw * ${sections[props.section.s][1]}`;
-} else {
-  sections.info[0] = true;
-}
 </script>
 
 <template>
-  <div v-if="tournament?.is_announced" class="flex min-h-[calc(100vh_-_6rem)] flex-col items-center">
+  <div v-if="tournament?.is_announced" class="min-h-[calc(100vh_-_6rem)]">
```

Comment:
This look strange on wide monitor, there is a big empty space when the page is smaller and unlike before, the image does not scale on the whole screen (which is better btw, :+1: )
Old : 
![image](https://github.com/user-attachments/assets/9e3571db-a1c1-4949-ad5f-0c0b3ffadcc5)

New :
![image](https://github.com/user-attachments/assets/5c0e67b4-4167-43c4-befe-ece4a32e5b82)